.github/workflows/classify-issue-complexity.yml

```
@@ -0,0 +1,119 @@
+name: Classify Issue Complexity
+
+# AI-driven complexity classification for intelligent model routing
+# Simple tasks → DeepSeek Chat ($0.14/$0.28)
+# Complex tasks → DeepSeek R1 ($0.30/$1.20)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)